alternator/executor.cc

```diff
@@ -1121,9 +1121,9 @@ static future<executor::request_return_type> create_table_on_shard0(tracing::tra
     auto group0_guard = co_await mm.start_group0_operation();
     auto ts = group0_guard.write_timestamp();
     std::vector<mutation> schema_mutations;
-    auto ksm = create_keyspace_metadata(keyspace_name, sp, gossiper, ts);
+    auto ksm = create_keyspace_metadata(keyspace_name, sp, gossiper, ts, tags_map);
     try {
-        schema_mutations = service::prepare_new_keyspace_announcement(sp.local_db(), ksm, ts);
+       schema_mutations = service::prepare_new_keyspace_announcement(sp.local_db(), ksm, ts);
```

Comment:
This is a code style regression

Reply:
Sorry, I don't know what happened here, I'll fix it.

Reply:
disk corruption?

Reply:
More likely, a cat on my keyboard + me too lazy to read the patch before sending it :-(

Reply:
Fixed.